assets/dfn-panel.css

```diff
@@ -84,3 +84,7 @@ dfn {
   max-height: 30vh;
   overflow: auto;
 }
+
+.dfn-panel.hidden {
```

Comment:
There is an attribute named [`hidden`](https://html.spec.whatwg.org/multipage/interaction.html#the-hidden-attribute) which does the same 👀

Reply:
How can I forget!